react/features/toolbox/components/web/Toolbox.js

```diff
@@ -924,6 +1001,88 @@ class Toolbox extends Component<Props> {
         ];
     }
 
+    /**
+     * Renders a list of buttons that are moved to the overflow menu.
+     *
+     * @private
+     * @param {Array<string>} movedButtons - The names of the buttons to be
+     * moved.
+     * @returns {Array<ReactElement>}
+     */
+    _renderMovedButtons(movedButtons) {
+        const {
+            _chatOpen,
+            _raisedHand,
+            t
+        } = this.props;
+
+        return movedButtons.map(buttonName => {
+            switch (buttonName) {
+            case 'desktop':
+                return this._renderDesktopSharingButton(true);
+            case 'raisehand':
+                return (
+                    <OverflowMenuItem
+                        accessibilityLabel =
+                            { t('toolbar.accessibilityLabel.raiseHand') }
+                        icon = { 'icon-raised-hand' }
+                        key = 'raisedHand'
+                        onClick = { this._onToolbarToggleRaiseHand }
+                        text = {
+                            t(`toolbar.${
+                                _raisedHand
+                                    ? 'lowerYourHand' : 'raiseYourHand'}`
+                            )
+                        } />
+                );
+            case 'chat':
+                return (
+                    <OverflowMenuItem
+                        accessibilityLabel =
+                            { t('toolbar.accessibilityLabel.chat') }
+                        icon = { 'icon-chat' }
+                        key = 'chat'
+                        onClick = { this._onToolbarToggleChat }
+                        text = {
+                            t(`toolbar.${
+                                _chatOpen ? 'closeChat' : 'openChat'}`
+                            )
+                        } />
+                );
+            case 'closedcaptions':
+                return <ClosedCaptionButton showLabel = { true } />;
+            case 'info':
+                return <InfoDialogButton showLabel = { true } />;
+            case 'invite':
+                return (
+                    <OverflowMenuItem
+                        accessibilityLabel =
+                            { t('toolbar.accessibilityLabel.invite') }
+                        icon = 'icon-invite'
+                        key = 'invite'
+                        onClick = { this._onToolbarOpenInvite }
+                        text = { t('toolbar.invite') } />
+                );
```

Comment:
Do you want to go ahead and add tooltips to all these using OverflowMenuItem? Right now some will have them and others won't. The perpetual-to-do of moving all buttons to use AbstractButton would have solved this but might or might not be more effort.

Reply:
Even before adding this logic there were some buttons that have tooltips and others don't have. IMHO we should remove the tooltips from the buttons in the overflow menu.

I'll prefer to solve the problem for all buttons in the overflow menu trough another PR. I guess it worth discussing it internally.